tools/ide_plugins/pysa-vscode/src/main.ts

```diff
@@ -39,7 +39,7 @@ export async function activate(_: vscode.ExtensionContext) {
     };
     
     let clientOptions: LanguageClientOptions = {
-        documentSelector: [{scheme: 'file', language: 'python'}],
```

Comment:
Given that @gracewgao's changes have landed, you can probably use the `pysa_model` language from #426